--- liblinphone-5.2.51/src/CMakeLists.txt.orig	2023-04-20 20:45:01.892542923 +0200
+++ liblinphone-5.2.51/src/CMakeLists.txt	2023-04-20 20:46:02.122216631 +0200
@@ -32,7 +32,7 @@ set(PUBLIC_LINK_LIBS
 )
 
 if (ENABLE_FLEXIAPI)
-	list(APPEND LINK_LIBS jsoncpp_static)
+	list(APPEND LINK_LIBS jsoncpp_lib)
 endif()
 
 #APPLE_LIBS is mandatory for apple compilation
@@ -650,11 +650,7 @@ if(ENABLE_STATIC)
 	)
 
 	if (ENABLE_FLEXIAPI)
-		if (XCODE)
-			target_link_libraries(linphone-static PRIVATE jsoncpp_static)
-		else()
-			target_link_libraries(linphone-static PRIVATE jsoncpp_object)
-		endif()
+			target_link_libraries(linphone-static PRIVATE jsoncpp_lib)
 	endif()
 
 	if (APPLE)
@@ -699,11 +695,7 @@ if(ENABLE_SHARED)
 	)
 
 	if (ENABLE_FLEXIAPI)
-		if (XCODE)
-			target_link_libraries(linphone PRIVATE jsoncpp_static)
-		else()
-			target_link_libraries(linphone PRIVATE jsoncpp_object)
-		endif()
+			target_link_libraries(linphone PRIVATE jsoncpp_lib)
 	endif()
 
 	if(APPLE)
--- liblinphone-5.2.51/tester/CMakeLists.txt.orig	2023-04-21 18:51:54.676272194 +0200
+++ liblinphone-5.2.51/tester/CMakeLists.txt	2023-04-21 18:51:57.524566770 +0200
@@ -322,6 +322,7 @@ set(SOURCE_FILES_CXX
 if(ENABLE_FLEXIAPI)
 	list(APPEND SOURCE_FILES_CXX account_creator_flexiapi_tester.cpp)
 	list(APPEND SOURCE_FILES_CXX flexiapiclient-tester.cpp)
+	list(APPEND OTHER_LIBS_FOR_TESTER jsoncpp_lib)
 endif()
 
 if(ENABLE_ADVANCED_IM)
