--- liblinphone-5.1.73/src/CMakeLists.txt.orig	2023-04-16 20:23:18.411880391 +0200
+++ liblinphone-5.1.73/src/CMakeLists.txt	2023-04-17 17:01:13.969495158 +0200
@@ -31,11 +31,7 @@ set(PUBLIC_LINK_LIBS
 )
 
 if (ENABLE_FLEXIAPI)
-	if (XCODE)
-		list(APPEND LINK_LIBS jsoncpp_static)
-	else()
-		list(APPEND LINK_LIBS jsoncpp_object)
-	endif()
+		list(APPEND LINK_LIBS jsoncpp_lib)
 endif()
 
 #APPLE_LIBS is mandatory for apple compilation
@@ -618,11 +614,7 @@ if(ENABLE_STATIC)
 	)
 
 	if (ENABLE_FLEXIAPI)
-		if (XCODE)
-			target_link_libraries(linphone-static PRIVATE jsoncpp_static)
-		else()
-			target_link_libraries(linphone-static PRIVATE jsoncpp_object)
-		endif()
+			target_link_libraries(linphone-static PRIVATE jsoncpp_lib)
 	endif()
 
 	if (APPLE)
@@ -673,11 +665,7 @@ if(ENABLE_SHARED)
 	)
 
 	if (ENABLE_FLEXIAPI)
-		if (XCODE)
-			target_link_libraries(linphone PRIVATE jsoncpp_static)
-		else()
-			target_link_libraries(linphone PRIVATE jsoncpp_object)
-		endif()
+			target_link_libraries(linphone PRIVATE jsoncpp_lib)
 	endif()
 
 	if(APPLE)
--- liblinphone-5.1.73/tester/CMakeLists.txt.orig	2022-12-06 10:47:26.000000000 +0100
+++ liblinphone-5.1.73/tester/CMakeLists.txt	2023-04-19 16:37:14.557171234 +0200
@@ -31,11 +31,7 @@ set(OTHER_LIBS_FOR_TESTER
 )
 
 if (ENABLE_FLEXIAPI)
-	if (XCODE)
-		list(APPEND OTHER_LIBS_FOR_TESTER jsoncpp_static)
-	else()
-		list(APPEND OTHER_LIBS_FOR_TESTER jsoncpp_object)
-	endif()
+		list(APPEND OTHER_LIBS_FOR_TESTER jsoncpp_lib)
 endif()
 
 if(INTL_FOUND)
