--- liblinphone-4.5.24/wrappers/cpp/CMakeLists.txt.orig	2021-07-06 10:06:15.000000000 +0200
+++ liblinphone-4.5.24/wrappers/cpp/CMakeLists.txt	2021-07-15 21:33:45.817747269 +0200
@@ -38,6 +38,7 @@
 )
 add_custom_target(linphone-cxx-wrapper ALL DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/include/linphone++/linphone.hh" "${CMAKE_CURRENT_BINARY_DIR}/src/linphone++.cc")
 
+if(ENABLE_SHARED)
 add_library(linphone++ SHARED
 	object.cc
 	tools.cc
@@ -81,6 +82,42 @@
 	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 	FRAMEWORK DESTINATION Frameworks
 )
+endif()
+if(ENABLE_STATIC)
+add_library(linphone++-static STATIC
+	object.cc
+	tools.cc
+	${CMAKE_CURRENT_BINARY_DIR}/src/linphone++.cc
+)
+add_dependencies(linphone++-static linphone-cxx-wrapper)
+
+if(APPLE)
+	set_target_properties(linphone++-static PROPERTIES OUTPUT_NAME linphone++)
+else()
+	set_target_properties(linphone++-static PROPERTIES OUTPUT_NAME liblinphone++)
+endif()
+set_target_properties(linphone++-static PROPERTIES PREFIX "")
+target_link_libraries(linphone++-static
+	PRIVATE ${BCTOOLBOX_CORE_LIBRARIES} ${BELLESIP_LIBRARIES} linphone
+)
+
+target_include_directories(linphone++-static
+	PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/include
+	PRIVATE ${PROJECT_BINARY_DIR}/include
+	PRIVATE ${PROJECT_SOURCE_DIR}/include
+	PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
+	PRIVATE ${BCTOOLBOX_INCLUDE_DIRS}
+	PRIVATE ${BELLESIP_INCLUDE_DIRS}
+)
+
+install(TARGETS linphone++-static EXPORT LinphoneCxxTargets
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	FRAMEWORK DESTINATION Frameworks
+)
+endif()
+
 install(FILES object.hh
 	DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/linphone++
 )
