--- liblinphone-5.3.32/cmake/LibLinphoneConfig.cmake.in.orig	2024-03-09 13:44:49.000000000 +0100
+++ liblinphone-5.3.32/cmake/LibLinphoneConfig.cmake.in	2024-04-01 07:13:16.219267313 +0200
@@ -60,6 +60,9 @@ endif()
 if(@Lime_FOUND@)
 	find_dependency(Lime)
 endif()
+if(@ENABLE_FLEXIAPI@)
+	find_dependency(jsoncpp)
+endif()
 if(@BUILD_SHARED_LIBS@)
 else()
 	list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
