--- liblinphone-5.3.32/src/factory/factory.cpp.orig	2024-03-10 13:08:36.628399613 +0100
+++ liblinphone-5.3.32/src/factory/factory.cpp	2024-03-10 13:10:03.947926563 +0100
@@ -849,7 +849,7 @@ LinphoneContent *Factory::createQRCode(c
 	}
 
 	auto writer = ZXing::MultiFormatWriter(format).setMargin((int)margin).setEncoding(encoding).setEccLevel(eccLevel);
-	auto matrix = writer.encode(ZXing::TextUtfEncoding::FromUtf8(code), (int)width, (int)height);
+	auto matrix = writer.encode(code, (int)width, (int)height);
 	auto bitmap = ZXing::ToMatrix<uint8_t>(matrix);
 
 	LinphoneContent *content = Factory::createContent();
