--- liblinphone-5.1.73/src/CMakeLists.txt.orig	2023-04-13 16:13:26.000000000 +0200
+++ liblinphone-5.1.73/src/CMakeLists.txt	2023-04-16 18:53:47.034312975 +0200
@@ -617,9 +617,9 @@ if(ENABLE_STATIC)
 
 	if (ENABLE_FLEXIAPI)
 		if (XCODE)
-			target_link_libraries(linphone PRIVATE jsoncpp_static)
+			target_link_libraries(linphone-static PRIVATE jsoncpp_static)
 		else()
-			target_link_libraries(linphone PRIVATE jsoncpp_object)
+			target_link_libraries(linphone-static PRIVATE jsoncpp_object)
 		endif()
 	endif()
 
